templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,13 +1,16 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.3</feature>
+        <feature>servlet-4.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <feature>mpMetrics-2.3</feature>
+        <feature>mpHealth-2.2</feature>
+        <feature>mpConfig-1.4</feature>
     </featureManager>
     <quickStartSecurity userName="admin" userPassword="adminpwd"/>
     <keyStore id="defaultKeyStore" location="key.jks" type="jks" password="mpKeystore"/>
-    <httpEndpoint host="*" httpPort="${default.http.port}" 
-        httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
+    <httpEndpoint host="*" httpPort="${DEFAULT_HTTP_PORT}"
```

Comment:
Not sure if the "translation" works in this direction.. I know if you do "${default.http.port}" you can use an env var of "DEFAULT_HTTP_PORT".   Best to go back to "${default.http.port}"